CRM/Utils/Date.php

```diff
@@ -800,17 +800,20 @@ public static function getRange($startDate, $endDate) {
    * @return array
    *   start date, end date
    */
-  public static function getFromTo($relative, $from, $to) {
+  public static function getFromTo($relative, $from, $to, $fromTime = NULL, $toTime = NULL) {
+    if (empty($toTime)) {
```

Comment:
Would this be cleaner in the function signature - ie
public static function getFromTo($relative, $from, $to, $fromTime = 0, $toTime = '235959') {